### PR TITLE
Add blogpost about Vert.x native image generation.

### PR DESCRIPTION
**Pull request by Jotschi**

### CODE REVIEW

File: src/site/blog/posts/2018-05-31-vertx-native.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+I this blog post I would like to give you a preview on native image generation of Vert.x applications using GraalVM.
+
+With [GraalVM](https://www.graalvm.org/) it is possible to generate native executables.
+These executables can be directly run without the need of an installed JVM.
+
+
+## Benefits
+* The start up time is way faster. It is no longer required to wait for the start up of the JVM. The application is usually up and running in a matter of milliseconds.
+
+* Reduced Memory. I measured 40 MB memory usage (RSS) for the Vert.x Web application which I'm going to showcase.
```
Comment (vietj) [inline src/site/blog/posts/2018-05-31-vertx-native.md]: _Reduced memory footprint_

File: src/site/blog/posts/2018-05-31-vertx-native.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+			.listen(8080);
+
+	}
+
+}
+```
+
+## GraalVM
+
+GraalVM runs a static analysis on the generated application in order to find the reachable code.
```
Comment (vietj) [inline src/site/blog/posts/2018-05-31-vertx-native.md]: link to graal vm web site

Comment (Jotschi) [reply src/site/blog/posts/2018-05-31-vertx-native.md]: Added links

File: src/site/blog/posts/2018-05-31-vertx-native.md (hunk 1 above)
Comment (Jotschi) [reply src/site/blog/posts/2018-05-31-vertx-native.md]: Updated section.

**Comment by vietj** [comment]
looks great, I'll make another review over the week-end to publish it next early next week

**Pull request merged**